src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -252,32 +252,36 @@
     @Test
     public void testsTarget() throws Exception {
         final Path output = Files.createTempDirectory("");
-        new XslReport(
-            new Skeleton(new FakeBase(this.target)).xml(),
-            this.metric, new XslCalculus()
-        ).save(output);
-        final String xpath;
-        if (Double.isNaN(this.value)) {
-            xpath = "//class[@id='%s' and @value='NaN']";
-        } else {
-            xpath = "//class[@id='%s' and number(@value)=%.4f]";
-        }
-        new Assertion<>(
-            new FormattedText(
-                "Must exists with target '%s' and value '%s'",
-                this.target, this.value
-            ).asString(),
-            XhtmlMatchers.xhtml(
-                new TextOf(
-                    output.resolve(String.format("%s.xml", this.metric))
-                ).asString()
-            ),
-            XhtmlMatchers.hasXPaths(
-                String.format(
-                    xpath,
+        try {
```

Comment:
@dgarus  Can't we use same approach here? Is it related to the fact that it is JUnit4 parametrized test?

Reply:
Yes, You are right.
I got the exception:
`java.lang.Exception: Method testsTarget should have no parameters`

If I try through the instance field it doesn't initiate it.

Reply:
@dgarus Please resolve conflicts. The master branch has JUnit 5 parametrized tests now (you may have to add an additional arg to the method). May be it is possible to apply the same approach. If it has the same problem, it is ok for me to create and delete explicitly as you did.

Reply:
@HDouss I've merged master to branch. We don't have this problem anymore.